src/babel/transformation/transformers/es6/arrow-functions.js

```diff
@@ -1,4 +1,28 @@
+/**
```

Comment:
In the future each transformer will be split into a separate package so the `README.md` of each of them can be what the site uses.